packages/core-mobile/app/services/send/SendServiceAVM.ts

```diff
@@ -58,12 +59,6 @@ export class SendServiceAVM {
             sendFee
           }
 
-          if (!address)
-            return SendServiceAVM.getErrorState(
-              newState,
-              SendErrorMessage.ADDRESS_REQUIRED
-            )
-
```

Comment:
wasn't it better with specific error message?  does it show `address required` error now when address is missing?

Reply:
QA was complaining about it since it was not a good UX